### DISCUSSION TITLE
"Autoplay is not defined"

### DISCUSSION THREAD

**Discussion by missusmy** [davidjerleke/embla-carousel#258] [Help] [answered]
I get this console error when I try to implement autoplay with a link to the script on cdn. Please help.

**Comment by davidjerleke** [comment, marked as answer]
Hi @missusmy,

As shown in the [docs](https://www.embla-carousel.com/get-started/cdn/#adding-plugins), plug-ins that are used with a CDN is prefixed with `EmblaCarousel<plugin-name>`. I guess this piece of information could be added to each plug-in page too.

So try using `EmblaCarouselAutoplay` instead of `Autoplay`.

Please mark this answer as correct if it solves the problem.

Best,
David

**Reply by missusmy** [reply]
Thanks! That worked for the autoplay.

Off topic, is there documentation on how to update our script from 5.0.1 to 6.0.x? There seems to be several things that aren't supported anymore.

**Reply by davidjerleke** [reply]
@missusmy please take a moment to read the [release notes](https://github.com/davidcetinkaya/embla-carousel/releases/tag/v6.0.0) for v6.0.0. These should cover all breaking changes. 

Let me know if helps.

Best,
David

**Reply by missusmy** [reply]
Okay, thanks.
